docker/grafana/provisioning/dashboards/lodestar.json

```diff
@@ -4162,6 +4162,409 @@
+            },
+            "tooltip": {
+              "mode": "single"
+            }
+          },
+          "pluginVersion": "8.0.0",
+          "targets": [
+            {
+              "exemplar": true,
+              "expr": "delta(beacon_fork_choice_changed_head_total[$__rate_interval])",
+              "hide": false,
+              "interval": "",
+              "legendFormat": "Changed Heads",
+              "refId": "A"
+            },
+            {
+              "exemplar": true,
+              "expr": "delta(beacon_fork_choice_requests_total[$__rate_interval])",
+              "hide": false,
+              "interval": "",
+              "legendFormat": "Total Requests",
+              "refId": "B"
+            }
+          ],
+          "title": "Total findHead Requests vs changed heads",
+          "type": "timeseries"
+        },
+        {
+          "datasource": null,
+          "fieldConfig": {
+            "defaults": {
+              "color": {
+                "mode": "palette-classic"
+              },
+              "custom": {
+                "axisLabel": "",
+                "axisPlacement": "auto",
+                "barAlignment": 0,
+                "drawStyle": "line",
+                "fillOpacity": 0,
+                "gradientMode": "none",
+                "hideFrom": {
+                  "legend": false,
+                  "tooltip": false,
+                  "viz": false
+                },
+                "lineInterpolation": "linear",
+                "lineWidth": 1,
+                "pointSize": 5,
+                "scaleDistribution": {
+                  "type": "linear"
+                },
+                "showPoints": "auto",
+                "spanNulls": false,
+                "stacking": {
+                  "group": "A",
+                  "mode": "none"
+                },
+                "thresholdsStyle": {
+                  "mode": "off"
+                }
+              },
+              "mappings": [],
+              "thresholds": {
+                "mode": "absolute",
+                "steps": [
+                  {
+                    "color": "green",
+                    "value": null
+                  },
+                  {
+                    "color": "red",
+                    "value": 80
+                  }
+                ]
+              }
+            },
+            "overrides": []
+          },
+          "gridPos": {
+            "h": 8,
+            "w": 12,
+            "x": 0,
+            "y": 10
+          },
+          "id": 132,
+          "options": {
+            "legend": {
+              "calcs": [],
+              "displayMode": "hidden",
+              "placement": "bottom"
+            },
+            "tooltip": {
+              "mode": "single"
+            }
+          },
+          "targets": [
+            {
+              "exemplar": true,
+              "expr": "12*rate(beacon_fork_choice_find_head_seconds_count[$__rate_interval])",
```

Comment:
This does not represent utilization but an approximation of the number of times find head is called per slot.

To do a proper utilization rate you have to take the rate of the sum metric without any multiplier `rate(_sum[$__rate_interval])`

Reply:
corrected the heading, added a new utilization graph